src/sentry/services/hybrid_cloud/organization/impl.py

```diff
@@ -464,3 +464,18 @@ def update_region_user(self, *, user: RpcRegionUser, region_name: str) -> None:
         # the attribute we are changing never needs to produce an outbox.
         with in_test_psql_role_override("postgres"):
             OrganizationMember.objects.filter(user_id=user.id).update(user_is_active=user.is_active)
+
+    def get_option(self, *, organization_id: int, key: str) -> OptionValue:
+        orm_organization = Organization.objects.get(id=organization_id)
```

Comment:
```suggestion
        orm_organization = Organization.objects.get_from_cache(id=organization_id)
```
Option reads don't use much organization state so we should be able to read those records from cache.